### ISSUE TITLE
[Concept Entry] Sklearn quadratic-regression-analysis

### ISSUE THREAD

**Issue by codecademy-docs** [Codecademy/docs#4824]
### Reason/inspiration (optional)

We would like a new entry on the `Quadratic Regression Analysis` concept under Sklearn. The entry should go in a new file under `docs/content/sklearn/concepts/quadratic-regression-analysis/quadratic-regression-analysis.md`.

The entry should include:

- An introduction to the concept
- A ‘Syntax’ section that provides the syntax for the concept (if applicable)
- An ‘Example’ section that provides an example demonstrating the concept in use
- A ‘Codebyte Example’ section that provides a codebyte example demonstrating the concept in use

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Concept Entrty for Existing Topic

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

**Comment by Dev-29** [comment]
I would like to work on this.

**Comment by letsbuildanything** [comment]
@Dev-29  You’re assigned :tada:  In addition to the documents linked in the description, please also look at the [Contribution Guide](https://www.codecademy.com/resources/docs/contribution-guide).  After creating a PR, the maintainer(s) (with the collaborator label) will add comments/suggestions to address any revisions prior to approval.
Is this your first contribution to Codecademy Docs? If so, we’re curious to know how you found out about contributing to Docs.

**Comment by Dev-29** [comment]
Hi, I'm having a poor network where I am, so I'm having trouble cloning the entire repo. I'm working on it, and it will be done within this week. 

Thank you

**Comment by uhck** [comment]
I can take this issue. Thanks!

**Comment by uhck** [comment]
Is this performed using PolynomialFeatures with LinearRegression in Scikit-Learn?

**Comment by mamtawardhani** [comment]
Hey @uhck! Yes, in Scikit-Learn, quadratic regression is performed using the combination of PolynomialFeatures and LinearRegression.

**Comment by mamtawardhani** [comment]
Also, @uhck You’re assigned 🎉 In addition to the documents linked in the description, please also look at the Contribution Guide. After creating a PR, the maintainer(s) (with the collaborator label) will add comments/suggestions to address any revisions before approval.
